fleet_vehicle_inspection/tests/test_fleet_vehicle_inspection.py

```diff
@@ -79,3 +80,30 @@ def test_fleet_vehicle_inspection(self):
         # inspection2 confirm without items
         with self.assertRaises(UserError):
             self.inspection2.button_confirm()
+
+    def test_fleet_vehicle_inspection_with_amount(self):
+        test_vendor = self.env["res.partner"].create({"name": "test vendor"})
+        self.inspection.inspection_line_ids[0].action_item_success()
+        self.inspection.inspection_line_ids[1].action_item_success()
+        self.inspection._compute_inspection_result()
+        self.inspection.amount = 100
+        self.inspection.vendor_id = test_vendor.id
+        with self.assertRaises(UserError):
```

Comment:
```suggestion
        with self.assertRaisesRegex(UserError, "Inspection cannot be completed"):
```

💡 tip: Always use `assertRaisesRegex` over `assertRaises`, this way you are sure to be asserting the exact exception you expect. UserErrors can be arised from many different places, and so you risk to be testing something you don't want

⚠️  Also, by the look of the PR diff, it kinda looks like you were expecting a `ValidationError(msg="Must select service type")` here, not a `UserError` 🤔